frontend/packages/dev-console/integration-tests/README.md

```diff
@@ -41,48 +88,163 @@ frontend/packages/dev-console/integration-tests/
 |   |    └──helm-navigation.feature
 |   ├──  BestPractices.md   <--- Gherkin script standards
 ├── fixtures                <--- Test data required for scripts
+├── testData                <--- Test data required for scripts
 ├── cypress.json            <--- cypress configuration file
-
 ```
 
 ## This file consists of guide lines to create automation scripts
+## Designing Gherkin scripts
+
+### Feature files
+
+1. Below practices helpful to design the gherkin scripts
+   [Best Practices for writing gherkin scripts](frontend/packages/dev-console/integration-tests/features/BestPractices.md)
+2. Use the same repo for writing gherkin scripts, So that installed plugins gives you the intellisense to select the existing steps [which helps to improve re-usability]
```

Comment:
```suggestion
2. Use the same repo for writing gherkin scripts, so that installed plugins gives you the intellisense to select the existing steps which helps to improve re-usability.
```